pdc/apps/compose/models.py

```diff
@@ -505,3 +505,41 @@ class Meta:
         unique_together = (
             ("variant_arch", "image"),
         )
+
+
+class Location(models.Model):
+    name = models.CharField(max_length=50)
+    short = models.CharField(max_length=50, unique=True)
+
+
+class Scheme(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+
+class ComposeTree(models.Model):
+    compose             = models.ForeignKey("Compose")
+    variant             = models.ForeignKey("Variant")
+    arch                = models.CharField(max_length=50)
```

Comment:
I think these should be one of currently defined arches and not just any string.